src/app/kbv/fields.js

```diff
@@ -9,4 +9,9 @@ module.exports = {
     items: ["stop", "continue"],
     validate: ["required"],
   },
+  "sa-income-from-pensions": {
```

Comment:
This is possible but gets messy having two different namings for one controller. I think self-assessment-router across here will be suitable. Otherwise code could get difficult to follow